packages/Webkul/Shop/src/Resources/lang/en/app.php

```diff
@@ -845,6 +845,17 @@
                 'password'                => 'Password',
             ],
 
+            'admin' => [
+                'new_customer_registration_notification' => [
+                    'dear'         => 'Dear Admin',
+                    'description'  => 'A new customer has registered in the system.',
+                    'name'         => 'Customer Name: :customerName',
```

Comment:
Change the key name **from** `new_customer_registration_notification` **to** 

![code-snapshot](https://github.com/bagisto/bagisto/assets/121479029/f9f7792f-ba93-4210-8c54-f8cbc2cc5bee)